akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -26,4 +28,96 @@ object Directives extends AllDirectives {
 
   @varargs override def getFromBrowseableDirectories(directories: String*): Route =
     super.getFromBrowseableDirectories(directories: _*)
+
+  /**
+   * Partially applying functions for the Java World!
```

Comment:
And about the comment about this is not the right place, I completely agree with you @jypma (this was done in an older version), Any idea where this could be placed?